compiler/rustc_query_system/src/query/plumbing.rs

```diff
@@ -467,17 +446,14 @@ where
     result
 }
 
-fn load_from_disk_and_cache_in_memory<CTX, K, V>(
+fn load_from_disk_and_cache_in_memory<CTX: QueryContext, K, V>(
     tcx: CTX,
     key: K,
     prev_dep_node_index: SerializedDepNodeIndex,
     dep_node_index: DepNodeIndex,
     dep_node: &DepNode<CTX::DepKind>,
     query: &QueryVtable<CTX, K, V>,
-) -> V
-where
-    CTX: QueryContext,
```

Comment:
All these changes moving bounds from where bounds to generic arg declarations are not necessary for this PR. Please revert them. If I remember correctly this is all on purpose, let's not change it in a drive-by manner.

Reply:
Should I also change it when there was no where clause but now the generic args are a lot longer?
Like
```
impl<D: Copy + Clone + Eq + Hash, Q: Clone> QueryJob<D, Q> {
```
to
```
impl<D, Q> QueryJob<D, Q>
where
    D: Copy + Clone + Eq + Hash,
    Q: Clone,
```